<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryTimeData</class>
 <widget class="QDialog" name="HistoryTimeData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="deviceTypeCombo">
        <item>
         <property name="text">
          <string>选择设备类型</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="deviceIDCombo">
        <item>
         <property name="text">
          <string>选择设备ID</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>开始时间：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDateTimeEdit" name="startTimeEdit"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>结束时间：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDateTimeEdit" name="endTimeEdit"/>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>加载数据</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="generateChartButton">
        <property name="text">
         <string>生成折线图</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QPushButton" name="exportCSVButton">
        <property name="text">
         <string>导出数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>数据ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数值</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
